src/Components/Subtitle/index.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+import './index.css';
+
+export default class Subtitle extends Component {
+  render(){
+    return(
+      <h2 className="subtitle">Subtitle: Sample text for subtitle under the title</h2>
```

Comment:
We need a way to pass text to this component. Can you remove the placeholder text from here and add a prop so that we can send text through?